_jestsetup.js

```diff
@@ -1,9 +1,9 @@
-import fs from 'fs';
-import path from 'path';
+import fs from 'node:fs';
+import { join } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { jest } from '@jest/globals';
 import fetchMock from 'jest-fetch-mock';
 
-jest.mock('path');
```

Comment:
@stefanbuck I couldn't get this to work with esm. I'm not sure if it doesn't work at all, or if there's a different way of doing it now. The way I got the test to pass was to use the posix version of join directly.